srcpkgs/python3-scipy/template

```diff
@@ -1,25 +1,52 @@
 # Template file for 'python3-scipy'
 pkgname=python3-scipy
-version=1.3.2
-revision=2
+version=1.5.0
+revision=1
 wrksrc="scipy-${version}"
 build_style=python3-module
-pycompile_module="scipy"
 make_check_args="--force"
-hostmakedepends="python3-setuptools python3-Cython gcc-fortran"
-makedepends="python3-devel python3-numpy lapack-devel"
+hostmakedepends="python3-setuptools python3-Cython python3-numpy gcc-fortran"
+makedepends="python3-devel python3-numpy
+ lapack-devel openblas-devel python3-pybind11"
 depends="python3-numpy"
 checkdepends="python3-nose"
 short_desc="Scientific library for Python3"
 maintainer="Alessio Sergi <al3hex@gmail.com>"
 license="BSD-3-Clause"
 homepage="https://scipy.org/scipylib/"
 distfiles="https://github.com/scipy/scipy/releases/download/v${version}/scipy-${version}.tar.xz"
-checksum=1d2f09bcb6c4b66a65d9f49d21fa065f5396c940edac8285b87947b8d21b55f8
+checksum=23baeaa18803d12d1abdff3f5c148b1085c2dc4028c6b8efce652dde2119b41c
 
-nocross="https://build.voidlinux.org/builders/aarch64_builder/builds/24733/steps/shell_3/logs/stdio"
+if [ -n "$CROSS_BUILD" ]; then
+	CFLAGS+=" -I${XBPS_CROSS_BASE}${py3_sitelib}/numpy/core/include "
+	LDFLAGS+=" -L${XBPS_CROSS_BASE}${py3_sitelib}/numpy/core/lib "
```

Comment:
Does the space on the end need to be there

Reply:
The template originally specified `LDFLAGS+="-shared"` with no leading space, so obviously somebody expected a trailing space in `LDFLAGS` at one point. However, because I added the leading space and am the only consumer in this template, the trailing spaces aren't necessary.